book.js

```diff
@@ -1,10 +1,106 @@
+const myLibrary = (JSON.parse(localStorage.getItem('savedLibrary')) == null) ? [] : JSON.parse(localStorage.getItem('savedLibrary'));
```

Comment:
You might simplify this a little: `const myLibrary = JSON.parse(localStorage.getItem("savedLibrary")) || [];`